### DISCUSSION TITLE
Option to disable archiving to waybackmachine?

### DISCUSSION THREAD

**Discussion by bormay** [ArchiveBox/ArchiveBox#853] [General]
Should there be an option to disable archiving to the waybackmachine?

**Comment by pirate** [comment]
There is. It's mentioned in 3 places in the readme, and in the config docs. 🤙

**Comment by jrkotrla** [comment]
Can I get some additional clarity please? I've added a 
SAVE_ARCHIVE_DOT_ORG = False

into the ArchiveBox.conf but it's still submitting new links to archive.org.  which is less than ideal. as then I have to manually select the archive method every single time.

**Reply by pirate** [reply]
please post your `ArchiveBox.conf` file contents

**Reply by jrkotrla** [reply]
all done using archivebox config set command. 

```
[SERVER_CONFIG]
SECRET_KEY = {xx}
PUBLIC_INDEX = False

[ARCHIVE_METHOD_TOGGLES]
SAVE_ARCHIVE_DOT_ORG = False
```

**Reply by pirate** [reply]
Can you screenshot where you see it submitting links to Archive.org? Also check the config is correct with `archivebox config --get SAVE_ARCHIVE_DOT_ORG`.

**Reply by jrkotrla** [reply]
```
archivebox@299dc9968bbb:/data$ archivebox config --get SAVE_ARCHIVE_DOT_ORG
[i] [2021-11-26 00:59:02] ArchiveBox v0.6.2: archivebox config --get SAVE_ARCHIVE_DOT_ORG
    > /data

SAVE_ARCHIVE_DOT_ORG=False

archivebox@299dc9968bbb:/data$ 
```
<img width="813" alt="Screen Shot 2021-11-25 at 8 10 27 PM" src="https://user-images.githubusercontent.com/11574593/143512499-6cd327b5-dc67-444d-a736-3869a8544e48.png">
but all of the links still show archive links. what am I doing wrong?

**Reply by pirate** [reply]
Oh the icon coloration isn't accurate for that method, that doesn't necessarily mean it's being saved to Archive.org. Can you check `archive/<timestamp>/index.json` and `archive/<timestamp>/archive.org.txt` to see if it's actually being saved.

**Reply by jrkotrla** [reply]
whew, there's no archive.org.txt saved since roughly when I made that
change, so thank you it does seem to be working properly. I didn't realize
that it still indicates a link to archive even if it doesn't submit the
link to archive.org  (moved reply)

**Comment by Jonirico11** [comment]
Am sorry im idon't now

Pada tanggal Jum, 26 Nov 2021 09.14, Nick Sweeting ***@***.***>
menulis:

**Comment by blogdemoi** [comment]
Hi,

I am having the same problem. I disabled saving to archive.org in the ArchiveBox.conf, however when archiving from the cli it still archives to archive.org.

`archivebox config --get SAVE_ARCHIVE_DOT_ORG` returns:

> dietpi@DietPi:~/archivebox$ docker compose run archivebox config --get SAVE_ARCHIVE_DOT_ORG
> [i] [2023-05-13 22:49:58] ArchiveBox v0.6.2: archivebox config --get SAVE_ARCHIVE_DOT_ORG
>     > /data
> 
> SAVE_ARCHIVE_DOT_ORG=False

Thank you.

**Reply by pirate** [reply]
Can you screenshot the log output showing that it's saving to archive.org? Also double check that `./archive/<timestamp>/archive.org.txt` actually is present and contains a URL, otherwise it's not actually saving to archive.org.

**Reply by blogdemoi** [reply]
Here is the screenshot, Archive.org doesn't appear here:
![UXSLLXZ](https://github.com/ArchiveBox/ArchiveBox/assets/83965802/7f889071-e4b2-43b3-ac9e-1e46a632daaf)


`./archive/<timestamp>/archive.org.txt` is not present (bottom right panel):
![JUeIaVd](https://github.com/ArchiveBox/ArchiveBox/assets/83965802/09cbc06f-c3c2-47c9-81e7-404f8cd949ec)

However, when I click on the Archive.org icon in my Archivebox web gui, I am redirected to the Archive.org page where I can see it was saved there: https://web.archive.org/web/20230515103547/https://www.iias.asia/the-review/portraiture-and-early-studio-photography-china-and-japan

**Reply by blogdemoi** [reply]
I tried from the GUI, without saving to Archive.org, and the same thing happened, it saved to Archive.org.

I didn't change anything in my Archivebox install/configuration except that I am now saving to a USB key, attached to my Pi.

**Reply by pirate** [reply]
Clicking the button in the UI takes you to the URL to "search for or save if not present" on archive.org.  It's not actually saved *until* you click that link.

**Reply by pirate** [reply]
As long as you don't see the .txt file or the logs indicating archivebox has submitted it to archive.org, it isn't actually saved there. Clicking the link makes archive.org aware that a URL that's not archive was requested by a user, and then they go and archive it and display it to you, so don't click the link unless you want them to save it.

**Reply by blogdemoi** [reply]
Ah... I didn't understand right then. Thank you for the clarification!

I won't click anymore. :-)
